torch/linalg/__init__.py

```diff
@@ -88,6 +88,43 @@
 Alias of :func:`torch.det`.
 """)
 
+slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
+linalg.slogdet(input) -> (Tensor, Tensor)
+
+Calculates the sign and natural logarithm of the absolute values of the determinants of the determinant(s)
+of a square matrix or batches of square matrices :attr:`input`.
+
+Supports input of ``float``, ``double``, ``cfloat`` and ``cdouble`` datatypes.
+
+.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+
+.. note:: For matrices that have zero determinant, this returns ``(0, -inf)``.
```

Comment:
Could we simplify this by saying, "Matrices with zero determinants are treated as having a sign of zero, even though the natural logarithm of zero is treated as negative infinity."?

Reply:
I think it makes it more complicated. This note together with "sign * exp(logabsdet)" is simple?

Reply:
OK, if you think this is simpler let's stick with it.